components/brave_shields/content/browser/ad_block_service.cc

```diff
@@ -209,6 +208,37 @@ base::Value::Dict AdBlockService::UrlCosmeticResources(
         }
       }
     }
+
+    base::Value::List* procedural_actions =
+        resources.FindList("procedural_actions");
+    // In standard blocking mode, drop procedural filters but otherwise keep
+    // action filters.
+    if (procedural_actions) {
+      base::Value::List::iterator it = procedural_actions->begin();
+      while (it < procedural_actions->end()) {
+        DCHECK(it->is_string());
```

Comment:
this DCHECK is kind of pointless because `GetString` fails with a `CHECK`, but maybe `GetIfString` is more appropriate here and then keep the DCHECK?

Reply:
updated to `GetIfString`